plugin/plugin.go

```diff
@@ -133,12 +104,16 @@ func (e *Ratelimit) Evaluate(r rules.RuleMetadata, tx rules.TransactionState) {
 
 	if totalEventsOccuredInPreviousWindow < e.MaxEvents {
 		e.Zones[zone_name][currentTimeInSecond]++
-		fmt.Println(e.Zones)
+		log.Println(e.Zones)
```

Comment:
Do you want to keep this?

Reply:
No in final release it should not be kept